dashboards/Insight/Home_Dashboard.json

```diff
@@ -1048,6 +1105,14 @@
             "fixedColor": "rgb(31, 120, 193)",
             "mode": "fixed"
           },
+          "displayName": "Nodes Overview",
+          "links": [
+            {
+              "targetBlank": true,
+              "title": "",
```

Comment:
Is empty title fix the problem with title attribute popups? Are we going to postpone this problem?

Reply:
![image](https://user-images.githubusercontent.com/90199600/172683329-5ddc590e-ac75-438e-a078-366182884dc4.png)
I mean this on hover.

Reply:
This field may be empty. Label take from "Display name" field.